### PR TITLE
Refactored Headcount/Afkcheck using common methods

### PR DESCRIPTION
**Pull request by dmansdman**
Refactored steps 1-3 in both StartHeadcount and StartAfkCheck into methods in Common.ts

Minor change: added specific dropdown for Exaltation Dungeons.

I recommend viewing changes in split mode rather than unified.

### CODE REVIEW

File: src/commands/raid-leaders/StartAfkCheck.ts [hunk 1]
```diff
@@ -1,32 +1,9 @@
...
-
-type DungeonSelectionType = {
-    section: ISectionInfo;
-    afkCheckChan: TextChannel;
-    cpChan: TextChannel;
-    raiderRole: Role;
-    dungeons: IDungeonInfo[];
-    omittedDungeons: IDungeonInfo[];
-};
+import {getAvailableSections, DungeonSelectionType, getSelectedSection, getSelectedDungeon} from "../../instances/Common";
```
Comment (dmansdman) [inline src/commands/raid-leaders/StartAfkCheck.ts:6]: The removed imports were all transferred into Common

File: src/commands/raid-leaders/StartAfkCheck.ts [hunk 2]
```diff
@@ -1,32 +1,9 @@
...
-import {AdvancedCollector} from "../../utilities/collectors/AdvancedCollector";
-import {StringUtil} from "../../utilities/StringUtilities";
-import {ArrayUtilities} from "../../utilities/ArrayUtilities";
-import {MessageUtilities} from "../../utilities/MessageUtilities";
 import {SlashCommandBuilder} from "@discordjs/builders";
 import {DungeonUtilities} from "../../utilities/DungeonUtilities";
-import {canManageRaidsIn, hasPermsToRaid} from "../../instances/Common";
-import {ButtonConstants} from "../../constants/ButtonConstants";
-
-type DungeonSelectionType = {
```
Comment (dmansdman) [inline src/commands/raid-leaders/StartAfkCheck.ts:22]: This type was also moved to Common and is imported above.

File: src/instances/Common.ts [hunk 3]
```diff
@@ -527,4 +544,238 @@ export function hasPermsToRaid(roleReqs: string[] | undefined, member: GuildMemb
     }
 
     return false;
+}
+
```
Comment (dmansdman) [inline src/instances/Common.ts:548]: Consider revising documentation for the three new methods.

**Review by ewang2002 — CHANGES_REQUESTED** [review]
See our Discord conversation

**Review by ewang2002 — APPROVED** [review]
LGTM

**Pull request merged**